assets/scss/_base.scss

```diff
@@ -146,10 +146,21 @@ main {
   }
 }
 
+.background-pattern {
+  background-color: #316DE6;
+  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='100%25' height='100%25' viewBox='0 0 1600 800'%3E%3Cg stroke='%23000' stroke-width='66.7' stroke-opacity='0.05' %3E%3Ccircle fill='%23316DE6' cx='0' cy='0' r='1800'/%3E%3Ccircle fill='%232f6ae0' cx='0' cy='0' r='1700'/%3E%3Ccircle fill='%232e67d9' cx='0' cy='0' r='1600'/%3E%3Ccircle fill='%232c64d3' cx='0' cy='0' r='1500'/%3E%3Ccircle fill='%232b61cd' cx='0' cy='0' r='1400'/%3E%3Ccircle fill='%23295ec6' cx='0' cy='0' r='1300'/%3E%3Ccircle fill='%23285bc0' cx='0' cy='0' r='1200'/%3E%3Ccircle fill='%232758ba' cx='0' cy='0' r='1100'/%3E%3Ccircle fill='%232555b4' cx='0' cy='0' r='1000'/%3E%3Ccircle fill='%232453ae' cx='0' cy='0' r='900'/%3E%3Ccircle fill='%232350a8' cx='0' cy='0' r='800'/%3E%3Ccircle fill='%23224da1' cx='0' cy='0' r='700'/%3E%3Ccircle fill='%23214a9b' cx='0' cy='0' r='600'/%3E%3Ccircle fill='%231f4795' cx='0' cy='0' r='500'/%3E%3Ccircle fill='%231e4490' cx='0' cy='0' r='400'/%3E%3Ccircle fill='%231d428a' cx='0' cy='0' r='300'/%3E%3Ccircle fill='%231c3f84' cx='0' cy='0' r='200'/%3E%3Ccircle fill='%231B3C7E' cx='0' cy='0' r='100'/%3E%3C/g%3E%3C/svg%3E");
+  background-attachment: fixed;
+  background-size: cover;
+}
+
```

Comment:
The design looks good. Is there a way to keep the weight/ratio of the pages in sync with the current design present?
![image](https://user-images.githubusercontent.com/86051118/212816434-078a24b9-b8b5-4a9e-8e38-c20a2979a556.png)

![image](https://user-images.githubusercontent.com/86051118/212816454-5a15e3ec-f965-47eb-a9ad-7d92cc25cd29.png)

Reply:
Sorry, i couldn't understand what you meant by weight/ratio of the pages.

Reply:
In short, Is there a way to maintain the size ratio same as it was before?

Reply:
@NitishKumar06 what size are you viewing the site in? as shown in the screenshots below, the size/ratio looks fine.

<img width="1427" alt="Screenshot 2023-01-25 at 12 42 27 AM" src="https://user-images.githubusercontent.com/59417546/214386428-0538524f-8503-4c4e-8500-b083d47d5cf3.png">

<img width="1438" alt="Screenshot 2023-01-25 at 12 42 37 AM" src="https://user-images.githubusercontent.com/59417546/214386461-3def49a8-38b5-410e-bfd4-6adb58c4867e.png">